<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>骨骼 socket 节点自动生成工具</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="topMargin">
    <number>30</number>
   </property>
   <item>
    <widget class="MTableView" name="Table_View">
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="MPushButton" name="Clear_BTN">
       <property name="text">
        <string>清空 Socket</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MPushButton" name="Socket_BTN">
       <property name="text">
        <string> 自动生成 Socket</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <widget class="QMenuBar" name="MenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>627</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="Import_CSV_Action"/>
    <addaction name="Export_CSV_Action"/>
    <addaction name="separator"/>
    <addaction name="Close_Action"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="Help_Action"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="Import_CSV_Action">
   <property name="text">
    <string>读取 CSV</string>
   </property>
  </action>
  <action name="Export_CSV_Action">
   <property name="text">
    <string>导出 CSV</string>
   </property>
  </action>
  <action name="Close_Action">
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
  <action name="Help_Action">
   <property name="text">
    <string>帮助文档</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MPushButton</class>
   <extends>QPushButton</extends>
   <header>dayu_widgets/push_button.h</header>
  </customwidget>
  <customwidget>
   <class>MTableView</class>
   <extends>QTableView</extends>
   <header>dayu_widgets/item_view.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
